new/authentication_config_file.md

```diff
@@ -0,0 +1,73 @@
+---
+RFC: unassigned
+Title: Authentication Configuration
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Authentication Configuration
+
+Currently, the Chef community maintains many tools to aid in the
+management and switching of Chef Server configuration options - for
+example `knife-block`. Any non-ruby tools must also implement their own
+configuration management, since our configuration files must be parsed
+as though they were ruby.
+It would be delightful to provide a single configuration file that was
+solely responsible for handling authentication options, and was in a
+language agnostic format, such as TOML.
+
+## Motivation
+
+    As an Operations Engineer,
+    I want to manage nodes on many Chef Servers,
+    so that I can interact with my whole estate.
+
+    As a tool developer,
+    I want to use the language of my choice,
+    so that I can effectively build delightful Chef features.
+
+## Specification
+
+A new file, `~/.chef/credentials`, will be supported by the chef client
+libraries, and other implementations. It is a TOML file, containing one
+or more sections, each associated with a profile.
+
+```
+[default]
+node_name = "barney"
```

Comment:
If we're making something new, can we support `client_name` as an alias for this and use that as primary in the docs?

Reply:
👍